client-js/app/elements/labrad-plot.ts

```diff
@@ -367,8 +367,18 @@ export class Plot extends polymer.Base {
 
     this.lastData = null;
 
+    this.dataLimits.xMin = 0;
+    this.dataLimits.xMax = 0;
+    this.dataLimits.yMin = 0;
+    this.dataLimits.yMax = 0;
```

Comment:
Should use `NaN` as the sentinel values when data limits need to be recomputed. Might be good to modify the test case to put some traces entirely above or below 0 to check this. Also, probably want to reset `zMin` and `zMax` to take care of the 2D case, and add a data_demo case for that too.